### PR TITLE
Add whitenoise support

### PR DESCRIPTION
**Pull request by comzeradd**
I just added the bare minimum configuration for whitenoise

### CODE REVIEW

**Comment by johngian** [comment]
Please fix flake8 errors.

**Comment by johngian** [comment]
Have you tested this locally? If yes, how?

**Comment by johngian** [comment]
This way you are forcing whitenoise to be the default handler for static files in our wsgi app. This forces our current deployment to use it and it's most probably going to fail.

File: remo/settings/base.py [hunk 1]
```diff
@@ -52,14 +52,15 @@
 ]
 
 MIDDLEWARE_CLASSES = (
+    'django.middleware.security.SecurityMiddleware',
```
Comment (akatsoulas) [inline remo/settings/base.py]: After PR #1305 we need to add whitenoise to base_env_vars file too.

**Comment by comzeradd** [comment]
Following @akatsoulas comment, now that we have a distinct base settings file for our cluster, no need for a separate wsgi file. I just included the relevant configuration to `base_env_vars` only.

**Pull request merged**